charts/consul/test/terraform/aks/variables.tf

```diff
@@ -27,7 +27,7 @@ variable "cluster_count" {
 }
 
 variable "tags" {
-  type        = map
+  type        = map(any)
```

Comment:
Terraform changes?

Reply:
yeah, removing them

Reply:
I have terraform fmt watcher, so we probably missed some formatting somewhere